devito/ops/types.py

```diff
@@ -0,0 +1,106 @@
+import sympy
+
+import devito.types.basic as basic
+
+from devito.tools import dtype_to_cstr
+
+
+class OpsAccessible(basic.Symbol):
```

Comment:
@btrb here
maybe this name leads to confusion.
what about something like `OpsKernelArg` or `OpsKernelParam`

(also, i´d vote for capital OPS like `OPSLalala` instead of `OpsLalala`)

Reply:
I'm not particularly fond of either suggestion. Maybe `OpsAccessBase` in line with `Indexed`